src/models/monitoringStandard.js

```diff
@@ -13,6 +13,14 @@ export default (sequelize, DataTypes) => {
        *  status: standardId -< MonitoringStandardLink.standardId
        */
 
+      MonitoringStandard.hasMany(models.ActivityReportObjectiveCitation, { foreignKey: 'citationId', as: 'activityReportObjectiveCitations' });
+      MonitoringStandard.belongsToMany(models.ActivityReportObjective, {
+        through: models.ActivityReportObjectiveCitation,
+        foreignKey: 'citationId',
+        otherKey: 'activityReportObjectiveId',
+        as: 'activityReportObjectives',
+      });
+
```

Comment:
These should be in the file for the linkage, not in the file being linked too.

Reply:
outdated.